### DISCUSSION TITLE
Framer motion latest versions(v4 & up) doesnt work with CRA (for modal,popover etc)

### DISCUSSION THREAD

**Discussion by rohantanwar0809** [chakra-ui/chakra-ui#3977] [General] [answered]
Transitions/Animations for stackable layers components such as modal ,popover etc does not work when using framer motion v4 & up in Chakra UI CRA starter template . 
Basically, when i tried using basic modal from  chakra ui , the component was not visible and  not rendering properly.
In case of popovers,  the popover appeared correctly but the close icon did not make it disappear. (The callbacks worked fine btw)
Im not sure what went wrong for me but I had a lot of time invested in finding the solution to this problem.
**The solution is use framer motion ver below 4.0 (e.g v3.10.6)**

Happy to hear everyone's thoughts about this.

**Comment by leonardomso** [comment, marked as answer]
This was a bug caused by `framer-motion`. 

 Update it to the `4.1.16` version and it should be fixed. 
 
 You can read more about it here #3994.
